bin/rucio

```diff
@@ -99,9 +100,10 @@ def setup_logger(logger):
                     color = '\033[32;1m'
                 elif levelno >= logging.DEBUG:
                     color = '\033[36;1m'
+                    format_str = '%(asctime)s\t%(levelname)s\t%(filename)s\t%(funcName)s\t%(message)s\033[0m'
```

Comment:
@TomasJavurek this will print the filename and funcname also for INFO

i don't think this is intended?

Reply:
It is not. We already discussed with @Martin that we don't do it like this. However, we have, I think, more ugly problem with the loggin, and that's the duplicate messages.

Reply:
but although logging.INFO >= logging.DEBUG evaluates to True, it wouldn't be used for the logging.INFO due to the elif clause.